static.config.js

```diff
@@ -61,7 +76,32 @@ export default {
         component: 'src/site/pages/Home',
         getData: () => ({
           supportedAssets
-        })
+        }),
+        children: generateCombinationsFromArray(supportedAssets, 'symbol', '-').map(pair => {
```

Comment:
Some of these pairs will be invalid. For a pair X_Y make sure X has `deposit === true` and Y has `receive === true`